web/app/js/components/PodDetail.jsx

```diff
@@ -128,7 +128,7 @@ export default class PodDetail extends React.Component {
   }
 
   render() {
-    if(!this.state.loaded){
+    if(!this.state.loaded) {
```

Comment:
Ooh, is there a rule that we can use to make this `if (!this.state.loaded) {`? Maybe "space-after-keywords: 1"? I think I'd like to add that if it's not too intrusive.

Reply:
Ah, looks like that rule is deprecated. It should be:

```
"keyword-spacing": 1
```

And I think it would work well for our setup if you don't mind adding as part of this branch.

Reply:
totally!